packages/pyright-internal/src/languageService/signatureHelpProvider.ts

```diff
@@ -245,21 +249,28 @@ export class SignatureHelpProvider {
                 paramName = params[params.length - 1].name || '';
             }
 
-            parameters.push({
-                startOffset: label.length,
-                endOffset: label.length + paramString.length,
-                text: paramString,
-            });
+            const isKeywordOnly =
+                paramListDetails.params.find((param) => param.param.name === paramName)?.kind === ParamKind.Keyword;
```

Comment:
This check isn't quite correct. The problem is that this code finds only the first parameter with a matching name. Usually parameter names must be unique, but there are cases where you can have multiple parameters with the same name, and they can have different "kinds". I think this check would therefore be more robust: `paramListDetails.params.some(param => param.param.name === paramName && param.kind === ParamKind.Keyword).